source/index.html

```diff
@@ -1,12 +1,170 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="page-header">
+      <div class="page-header__wrapper">
+        <a class="logo page-header__logo">
+          <img src="img/logo-icon-mobile.svg" width="33" height="38" alt="Иконка логотипа cat energy">
+          <img src="img/logo.svg" width="101" height="18" alt="Логотип cat energy">
+        </a>
+      </div>
+      <button class="page-header__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <nav class="main-nav">
+        <ul class="main-nav__list">
+          <li class="main-nav__item">
+            <a class="main-nav__link main-nav__link--current" href="index.html">Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a class="main-nav__link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="main-nav__item">
+            <a class="main-nav__link" href="form.html">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <main class="page-main">
+      <h1 class="visually-hidden">Cat energy - магазин функционального питания для котов</h1>
 
+      <section class="promo">
```

Comment:
Обычно еще секцию на главной под меню называют [hero](https://www.awebco.com/blog/hero-section/)
```suggestion
      <section class="hero">
```